planning/behavior_path_planner/src/scene_module/avoidance/avoidance_module.cpp

```diff
@@ -616,28 +622,28 @@ void AvoidanceModule::updateEgoBehavior(const AvoidancePlanningData & data, Shif
     return;
   }
 
-  insertPrepareVelocity(path);
+  // insertPrepareVelocity(path);
```

Comment:
Oh...

Reply:
@tkimura4 @shmpwk Sorry, I fogot to remove `//` before merge. I create fix PR right now.

Reply:
https://github.com/autowarefoundation/autoware.universe/pull/4587